src/internal/views/nunjucks/view-licences/tabs/charge.njk

```diff
@@ -165,6 +165,8 @@
   <p>No agreements for this licence.</p>
 {% endif %}
 
-<a href="/licences/{{ licenceId }}/agreements/select-type" role="button" draggable="false" class="govuk-button govuk-button--secondary" data-module="govuk-button">
-  Set up a new agreement
-</a>
+{{ govukButton({
```

Comment:
Worth wrapping this in a conditional so that we don't allow the user to create an agreement for an inactive licence?

Same question for the Charge Version creation buttons above in the same file (charge.njk), do we want to wrap those button in a conditional?

Without a conditional around those buttons, I currently *can* create a charge version for a "dead licence". I assume that's not the desired effect, but I'm not too sure.

Reply:
I think this behaviour is OK.  E.g. they may need to add charge versions/agreements to a licence that has not yet started, or correct a problem with one that has recently expired